include/psp2/jpegencarm.h

```diff
@@ -1,101 +1,165 @@
 /**
  * \usergroup{SceJpegEncArm}
  * \internalusage{psp2/jpegencarm.h,SceJpegEncArm_stub,SCE_SYSMODULE_INTERNAL_JPEG_ENC_ARM}
+ *
+ * \note The maximum supported image width and height is 65520 for YCbCr420,
+ *       and 65528 for YCbCr422.\n\n
+ *       The width and height of YCbCr422 images must be a multiple of 8.
+ *       The width and height of YCbCr420 images must be a multiple of 16.
  */
 
 #ifndef _PSP2_JPEGENCARM_H_
 #define _PSP2_JPEGENCARM_H_
 
 #include <psp2/types.h>
-#include <psp2/jpegenc.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#define SCE_JPEGENCARM_MIN_COMP_RATIO     1   //!< Lowest compression ratio, best quality.
+#define SCE_JPEGENCARM_DEFAULT_COMP_RATIO 64  //!< Default compression ratio.
+#define SCE_JPEGENCARM_MAX_COMP_RATIO     255 //!< Highest compression ratio, lowest quality.
```

Comment:
@bythos14 here is 1 to 255. but a document of the `ratio` is 0 to 256.
what is right value?

Reply:
The documentation says **between** 0 and 256, so 0 and 256 are not included in that range of values.

Reply:
So 1 - 255 is the correct range

Reply:
in normally, `between a and b` is `a <= x <= b`. isn't it?

Reply:
Probably. I guess it would be better to the term inclusively here. I've made the adjustments.

Reply:
Documentation from where?

Reply:
The documentation written in the header.